main.js

```diff
@@ -0,0 +1,77 @@
+
+var board = {
+
+  state: [[5, 4, 3, 2, 1],[],[]],
+
+  moveDisc: function(move,target){
+  //move: base peg, target: target peg
```

Comment:
your code is not aligned properly, check your lint settings. This affects readability